### ISSUE TITLE
Error when revoking token. Zookeeper backend.

### ISSUE THREAD

**Issue by des4maisons** [hashicorp/vault#2218]
I am getting the following error when I revoke a token using a vault that is backed by zookeeper:

```
$ vault token-create
Key             Value
---             -----
token           49fcc486-4cbd-501f-f89b-726493097232
token_accessor  49a07002-280a-2035-505c-87f382e8ce7d
token_duration  0s
token_renewable false
token_policies  [root]

$ vault token-revoke 49fcc486-4cbd-501f-f89b-726493097232
Error revoking token: Error making API request.

URL: PUT http://localhost:8200/v1/auth/token/revoke
Code: 400. Errors:

* failed to revoke entry: failed to delete lease index entry: Removal of node `/vault/sys/expire/token/938fcfc12c56b6f24bbff0089fa02a37e2424d1d` failed: `zk: node does not exist`
```

I can reproduce this on a freshly init-ed vault 0.6.3 server, authenticating with the brand new root token.

I used this docker-compose setup to test from scratch: https://gist.github.com/des4maisons/12056e444849bfcac8a06051ee0bc222

**Comment by maxvt** [comment]
This is fixed in 0.6.4, specifically by https://github.com/hashicorp/vault/commit/461d2f3f27a54620e2773979add4422906f840e4.

**Comment by des4maisons** [comment]
@maxvt awesome, dunno how I managed to use an old version when explicitly trying to use newest! Thanks!
